Assignment1/command_line_interface.py

```diff
@@ -1,10 +1,126 @@
+from Assignment1.email_verifier import EmailVerifier
+from Assignment1.distance_formula import DistanceFormulaCalculator
+from Assignment1.retirement import year_can_retire
+from Assignment1.body_mass_index_calculator import BodyMassIndexCalculator
+
+
 class CommandLineInterface:
-    def __init__(self):
-        pass
 
-    def hello_world(self):
-        return "hello world"
+    @staticmethod
+    def get_commands():
+        return "commands: " \
+               "\n\temailval <email>" \
+               "\n\tbmi <feet> <inches> <weight>" \
+               "\n\tdist <x1> <y1> <x2> <y2>" \
+               "\n\tretire <age> <salary> <percent saving> <goal>" \
+               "\n\texit"
+
+    @staticmethod
+    def run_command(command):
+        """
+        Runs a given command and returns output as a string
+        :param command:
+        :return:
+        """
+
+        if isinstance(command, str) is False:
```

Comment:
Don't need to change, but a more "Pythonic" way to say this is `if not isinstance(command, str)`